extensions/window/tiling.lua

```diff
@@ -154,7 +153,7 @@ local function tileWindows(windows,rect,desiredAspect,processInOrder,preserveRel
         for _,win in ipairs(testWindows) do
           testFrame.h=win.area/testFrame.w
           local winAspectRatio=(testFrame.w/testFrame.h) * adjustAspect / desiredRowAspect
-          local __=geom(testFrame)
```

Comment:
This feels like it might have been deliberate? Maybe `geom()` is called here to tease out any errors in `testFrame`?